shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -753,6 +878,53 @@ enum RewriteTableTy {
     Peers,
 }
 
+fn get_prepared_result_message(message: &mut Message) -> Option<(CBytesShort, PreparedMetadata)> {
+    if let Some(Frame::Cassandra(CassandraFrame {
+        operation: CassandraOperation::Result(CassandraResult::Prepared(prepared)),
+        ..
+    })) = message.frame()
+    {
+        return Some((prepared.id.clone(), prepared.metadata.clone()));
+    }
+
+    None
+}
+
+fn get_execute_message(message: &mut Message) -> Option<(BodyReqExecuteOwned, CassandraMetadata)> {
+    if let Some(Frame::Cassandra(CassandraFrame {
+        operation: CassandraOperation::Execute(execute_body),
+        version,
+        stream_id,
+        tracing_id,
+        ..
+    })) = message.frame()
+    {
+        return Some((
+            *execute_body.clone(),
```

Comment:
We can avoid this clone like this:
![image](https://user-images.githubusercontent.com/5120858/193946764-a28ee5de-52ff-4768-95f8-c4ed00cbc99a.png)